app/src/main/java/chat/rocket/android/activity/AbstractAuthedActivity.java

```diff
@@ -36,6 +36,10 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     if (savedInstanceState == null) {
       handleIntent(getIntent());
     }
+    ConnectivityManager.getInstance(getApplicationContext()).keepAliveServer();
```

Comment:
Let's put this back into the `onResume`. It's trying to keep the connection alive.